Kernel/Devices/Generic/RandomDevice.cpp

```diff
@@ -40,10 +40,13 @@ ErrorOr<size_t> RandomDevice::read(OpenFileDescription&, u64, UserOrKernelBuffer
     });
```

Comment:
I would write this commit message like this:

```
Kernel: Implement write(2) to /dev/random as an entropy source

Instead of ignoring input from a write syscall on the /dev/random device
use the input as another source of entropy.

This stands in line with other Unix-like systems which allow their users
to increase overall entropy from userspace.
```